### ISSUE TITLE
Subnet docs are incorrect

### ISSUE THREAD

**Issue by ArtiomL** [F5Networks/f5-aws-cloudformation#50]
Hi,

The subnet documentation for `f5-existing-stack-across-az-cluster-byol-3nic-bigip.template` is wrong:

| CFT Label | Parameter Name | Required | Description |
| --- | --- | --- | --- |
| Subnet1 AZ1 | subnet1Az1 | Yes | Public or External subnet ID for Availability Zone 1. |
| Subnet1 AZ1 | subnet1Az2 | Yes | Public or External subnet ID for Availability Zone 2. |
| Subnet2 AZ1 | subnet1Az1 | Yes | Public or External subnet ID for Availability Zone 1. |
| Subnet2 AZ1 | subnet1Az2 | Yes | Public or External subnet ID for Availability Zone 2. |

Should probably be:

| CFT Label | Parameter Name | Required | Description |
| --- | --- | --- | --- |
| Subnet1 AZ1 | subnet1Az1 | Yes | Public or External subnet ID for Availability Zone 1. |
| Subnet1 AZ2 | subnet1Az2 | Yes | Public or External subnet ID for Availability Zone 2. |
| Subnet2 AZ1 | subnet2Az1 | Yes | Public or External subnet ID for Availability Zone 1. |
| Subnet2 AZ2 | subnet2Az2 | Yes | Public or External subnet ID for Availability Zone 2. |

Also it's unclear what's the different between Subnet1 and Subnet2 in each AZ as they have the same description.

Thanks.

**Comment by kskoh** [comment]
@ArtiomL Good catch on  the parameter name column. But in regards to the other one "Also it's unclear what's the different between Subnet1 and Subnet2 in each AZ as they have the same description.".  Not sure how to make it clearer in the description.  Subnet 1 in AZ1 and subnet1 in AZ2 are two different subnets.  Just naming the subnet number 1 and the other one number 2 but in two different AZs.  You have any suggestion to make it crystal clear to people reading it?  Your feedback is welcomed.

**Comment by JosephPJordan** [comment]
Sorry for the confusion, we are updating these docs to fix a number of issues in the 3NIC failover templates.  The table should look like:


CFT Label | Parameter Name | Required | Description
-- | -- | -- | --
Management Subnet AZ1 | managementSubnetAz1 | Yes | Management subnet ID for Availability Zone 1.
Management Subnet AZ2 | managementSubnetAz2 | Yes | Management subnet ID for Availability Zone 2.
Subnet1 AZ1 | subnet1Az1 | Yes | Public or External subnet ID for Availability Zone 1.
Subnet1 AZ2 | subnet1Az2 | Yes | Public or External subnet ID for Availability Zone 2.
Subnet2 AZ1 | subnet2Az1 | Yes | Private or Internal subnet ID for Availability Zone 1.
Subnet2 AZ1 | subnet2Az2 | Yes | Private or Internal subnet ID for Availability Zone 2.

The description at the top should read:
In a 3-NIC implementation, each BIG-IP VE has 3 network interfaces (NICs), one for management, one for external traffic, and one for internal traffic.

We are considering an out of band update to update the documentation for this issue, rather than waiting for the next release.  I'll update this issue when a decision is reached.

**Comment by crosbygw** [comment]
document edited

**Comment by C0missar** [comment]
When the documentation is updated, it would be helpful to indicate which of these three need to have a path to the Internet.  Originally, it was management, but in recent CFTs, it seems to require both management and external interfaces have a path to the Internet.

Also a comment that if either of those are private IPs, then the customer must provide a NAT gateway for access.

**Comment by crosbygw** [comment]
re-opening issue to have docs include additional notes suggested by @C0missar 

thanks!

**Comment by shyawnkarim** [comment]
This enhancement to our documentation is now being tracked internally with ID ESECLDTPLT-2055.

**Comment by shyawnkarim** [comment]
Closing. These legacy templates are now in maintenance mode and are being replaced by our next-generation templates available in the [Cloud Templates 2.0 GitHub repo](https://github.com/F5Networks/f5-aws-cloudformation-v2).

Our documentation has been completely redone for V2.
